internal/scorecard/plugins/resource_handler.go

```diff
@@ -129,7 +130,9 @@ func createFromYAMLFile(cfg BasicAndOLMPluginConfig, yamlPath string) error {
 			}
 		}
 		err = runtimeClient.Create(context.TODO(), obj)
-		if err != nil {
+		if errors.IsAlreadyExists(err) {
```

Comment:
Great 👍 

Missing just add changelog entry for the fix

Reply:
thanks, added the changelog entry.